index.js

```diff
@@ -11,7 +11,7 @@ var VirtualSerialPort = function(path, options, openImmediately, callback) {
     };
 
     if(openImmediately || openImmediately === undefined || openImmediately === null) {
-        process.nextTick(function() {
+        process.nextTick(() => {
```

Comment:
Hi @qdot!

Thank you very much for yout PR! Do you think you can fix this so we can merge and release it?